doc/overview.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Overview of dbxincluder
+=======================
+
+dbxincluder is an implementation of the XInclude 1.1 specification (https://www.w3.org/TR/xinclude-11)
+with support for DocBook transclusion (http://docbook.org/docs/transclusion).
+
+
+Limitations
+===========
+
+It does not support the XInclude 1.1 "xpointer" attribute, as the XPointer functionality is currently not exposed by lxml.
```

Comment:
> lxml does not expose that functionality to python code.

That might be even true, but that doesn't matter here. You are talking about *our* script which doesn't support XPointer ATM. The reason why it is so it's on a different page. ;-)